### ISSUE TITLE
vkreplay (screenshot layer) screenshots are too dark

### ISSUE THREAD

**Issue by eero-t** [LunarG/VulkanTools#153]
Setup:
- HW: SKL GT2
- OS: Ubuntu 16.04 with all the updates
- SW: Latest VulkanTools & Mesa from git

Use-case:
- Get a trace of the "cube" demo
- Get screenshot of the demo with vkreplay screenshot layer, and with external tool:
```
$ vkreplay -o cube.vktrace -s 4
^Z
[1]+  Stopped                 vkreplay -o cube.vktrace
$ import correct.png
$ fg
vkreplay -o cube.vktrace
^C
```

Expected result:
- vkreplay screenshot looks correct, like the one taken with external tool

Actual result:
- vkreplay screenshot is much darker than it should be

Notes:
- This issue happens with all the use-cases I've tried, not just with cube
- Validation layers don't show any errors (anymore, except for the mismatch in vkCreateSwapChainKHR() imageExtent & currentExtent)
- Maybe it's colorspace related issue? (there were earlier validation warnings about that)
- According to comment in issue 123, this is Intel specific

**Comment by ardaLunarG** [comment]
It seems like Screenshot layer converts all captures images to R8G8B8A8_UNORM. 
This color space is darker than sRGB (which Intel's default swapchain format)

This was intentional so captured frames on different machines can be compared to each other.

We are going to make a change and preserve color space. We will add parameter if user want screenshot on certain format it can be passed in.

Thus we change this bug to enhancement...

**Comment by ardaLunarG** [comment]
fixed within 0c65d4471b12

**Comment by eero-t** [comment]
Verified to work fine with Mesa/Intel now. Thanks!
